.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yaml

```diff
@@ -0,0 +1,23 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project 💡!
+labels: ["feature"]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for suggesting an idea for making Covalent better. Use this feature request template to suggest new functionality, performance improvements, or platforms we should support. Please search the issues in this repository to see if there is already a similar suggestion. If you do find something similar, please comment on that issue instead of opening a new one. If this is a question, please post it in the Q&A section of [GitHub Discussions](https://github.com/AgnostiqHQ/covalent/discussions) instead of creating an issue.
+
+  - type: textarea
+    attributes:
+      label: What should we add?
+      description: A clear and concise description of what you want to happen. If this is related to a problem, describe the problem you're encountering.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Describe alternatives you've considered
```

Comment:
(non-blocking) Perhaps "What alternatives have you considered?" or add a period at the end of "Describe alternatives ..."